reconcile/openshift_saas_deploy_trigger_base.py

```diff
@@ -269,7 +269,7 @@ def _trigger_tekton(
 
     target_ref = None
     if isinstance(spec, TriggerSpecConfig):
```

Comment:
You can remove the `isinstance` check since this param is available on all `spec` types.